gruntfile.js

```diff
@@ -145,6 +152,6 @@ module.exports = function(grunt) {
         grunt.task.run('copy');
     });
 
-    grunt.registerTask('serve', ['showcase', 'less:development', 'connect:livereload', 'openfin:serve', 'watch']);
+    grunt.registerTask('serve', ['showcase', 'less:development', 'connect:livereload', 'download', 'openfin:serve', 'watch']);
     grunt.registerTask('build', ['openfin:build', 'showcase']);
```

Comment:
Yes, we can have another task if you prefer. Now the zip download is every time.